packages/algob/test/builtin-tasks/compile.ts

```diff
@@ -95,8 +99,15 @@ describe("Compile task", () => {
   it("should recompile all files when --force is used", async () => {
     await op.resetAndCompile(true);
 
-    assert.equal(op.timestamp, 5);
-    assert.lengthOf(op.compiledFiles, 2);
-    assert.lengthOf(op.writtenFiles, 2);
+    assert.equal(op.timestamp, 9);
+    assert.lengthOf(op.compiledFiles, 4);
+    assert.lengthOf(op.writtenFiles, 4);
+  });
+
+  it("should return TEAL code", async () => {
+    const pyOp = new PyCompileOp(op);
+    const content = fs.readFileSync(path.join(ASSETS_DIR, f4), 'utf8');
+    const res = pyOp.compilePyTeal(f3) + '\n';
+    assert.deepEqual(content.toString(), res);
   });
```

Comment:
Please add a test for `compileOp.ensureCompiled(somePythonFile)` and check the returned `PyASCCache`.

Reply:
Ok, but for that `Algodv2` will be needed to compile the TEAL code, as `PyCompileOp` uses same `CompileOp` compiler functions for compiling Teal Files
should i create it ?

Reply:
Yes.

Reply:
ok

Reply:
This has one problem, we will need network config for creating `Algodv2`. 
we can take it from `AlgobRuntimeEnv` but we will need to have a network started before running tests else tests will fail.
is this ok ?
![algob](https://user-images.githubusercontent.com/26249903/97175774-04fd5800-17ba-11eb-8522-77ea82ec459c.png)

Reply:
network config will be different for different systems, will have to change it everytime we do test in different system